scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/utils/TaskKiller.java

```diff
@@ -36,15 +36,18 @@
 
 import org.apache.log4j.Logger;
 import org.objectweb.proactive.utils.Sleeper;
+import org.ow2.proactive.resourcemanager.utils.RMNodeStarter;
 
 public class TaskKiller {
 
     private static final Logger LOGGER = Logger.getLogger(TaskKiller.class);
 
-    private static final int SECONDS_TO_WAIT_UNTIL_AGGRESSIVELY_INTERRUPT_TASK = 10;
-    private static final int THREAD_INTERRUPT_SLEEP_TO_SECONDS_MULTIPLIER = 100;
+    private static final long CLEANUP_TIME_DEFAULT_SECONDS = 10;
     private static final long SLEEP_MILLISECONDS_DURING_THREAD_INTERRUPT = 10;
+    private static final long SLEEP_MILLISECONDS_BETWEEN_IS_THREAD_ALIVE_CHECK = 100;
+    private static final long MULTIPLIER_FROM_SECONDS_TO_ITERATION_WITH_CLEANUP_TIME_SLEEPER = 10;
```

Comment:
I think that is taste. I like to have the name in the code so that the developer reads immediately what is happening instead of wondering why dividing through 1000.